### ISSUE TITLE
Remove `-c, --[no-]checked` from CLI usage text

### ISSUE THREAD

**Issue by chalin** [dart-lang/pub#1922]
From https://github.com/dart-lang/pub/issues/1807#issuecomment-397573190:

The option `--[no-]checked` should be removed/replaced.

```console
> pub run -h
Run an executable from a package.

Usage: pub run <executable> [args...]
-h, --help            Print this usage information.
-c, --[no-]checked    Enable runtime type checks and assertions.

Run "pub help" to see global options.
```

cc @kevmoo @kwalrath

**Comment by kevmoo** [comment]
Maybe just add `--[no-]enable-asserts` and hit `--checked`?

**Comment by kevmoo** [comment]
Sorry. This is for pub client, not pkg:test – sorry for the fire drill @grouma

**Comment by natebosch** [comment]
> In Dart2 strong mode, the checked parameter only controls asserts, but not type checks.

https://api.dartlang.org/dev/2.0.0-dev.63.0/dart-isolate/Isolate/spawnUri.html

`--enable-asserts` would have the same implementation as `--checked` today. I'd be fine adding an alias for this and hiding the original.

**Comment by natebosch** [comment]
@kevmoo - what should the single character flag be?

**Comment by kevmoo** [comment]
`e`? `a`?

**Comment by chalin** [comment]
> `--enable-asserts` would have the same implementation as `--checked`

In the context of Dart 2, since the old and the new flags have the same effect, how about this: renaming the new flag to `--check-asserts` and reusing `-c` as a shorthand.

Of course `--no-checked` and `--checked` would be dropped.

**Comment by chalin** [comment]
Any update on this?

**Comment by kevmoo** [comment]
@jakemac53 @natebosch ?

**Comment by natebosch** [comment]
Fixed with 9f00679ef47bc79cadc18e143720ade6c06c0100
